src/layers/ImageLayer.js

```diff
@@ -63,7 +63,8 @@ const defaultProps = {
  */
 
 /**
- * @type {{ new(...props: LayerProps[]) }}
+ * @type {{ new <S extends string[]>(...props: import('../types').Viv<LayerProps, S>[]) }}
+ * @ignore
```

Comment:
adding `@ignore` makes it so the docs don't pick this up. It removes the duplication in the sidebar.

Reply:
You can think of this signature as _only_ being used for our emitted types, not our docs.